app/controllers/email_subscriptions_controller.rb

```diff
@@ -1,10 +1,10 @@
 class EmailSubscriptionsController < ApplicationController
   PREFERRED_EMAIL_NAME = {
-    email_digest_periodic: "#{ApplicationConfig['COMMUNITY_NAME']} digest emails",
+    email_digest_periodic: "#{SiteConfig.community_name} digest emails",
```

Comment:
I just did find and replace, but I suspect this may cause a problem in terms of `SiteConfig` being available yet here in this context, not entirely sure. Will check back on this tomorrow.

Reply:
Once the app is initialized you should be ok.

Reply:
It did indeed cause the error... `/home/travis/build/forem/forem/vendor/bundle/ruby/2.7.0/gems/rails-settings-cached-2.2.0/lib/rails-settings/base.rb:129:in `_all_settings': You cannot use settings before Rails initialize. (RuntimeError)`

I think this is because constants first get called before `SiteConfig` is initialized and available. So in these cases I needed to move it into a method.... It was just a couple instances where this was a problem.

Reply:
@benhalpern I don't fully understand what you did to solve this, can you clarify?